app/models/influencer.js

```diff
@@ -40,9 +41,14 @@ var Influencer = new Schema({
         timestamps: {
             createdAt: 'createdAt',
             updatedAt: 'updatedAt'
-        }
+        },
+        virtuals: true
     }
 );
 
+Influencer.virtual('fullName').get(function() {
```

Comment:
Yep you can add virtual setters too, though that can get trickier determining how to split the value, etc.